app/controllers/api/oceans_controller.rb

```diff
@@ -1,6 +1,9 @@
 class Api::OceansController < ApplicationController
-  def index(n_oceans = 4)
-    oceans = Array.new(n_oceans) do
+  def index
+    params = Rack::Utils.parse_query URI(request.url).query
+    n_oceans = params["n_oceans"]
+    n_oceans = "1" if n_oceans.empty?
```

Comment:
A better param name that follows general conventions would be  'limit'.
Also you can do : `params.fetch(:limit, 4)` while 4 is the default value then (I think we agreed on that not 1).